### ISSUE TITLE
Readme needs to be updated with the new version number.

### ISSUE THREAD

**Issue by ajhunyady** [infinyon/http-sink-connector#41]
Add `history` table to README, to give users visibility on the changes from one version to another.

**Comment by tjtelan** [comment]
Realistically, a changelog like this will only get updated if it is generated

**Comment by EstebanBorai** [comment]
> Realistically, a changelog like this will only get updated if it is generated

Yeah! I know we use `git cliff` for Fluvio repo, what do you think about doing the same here?
Thinking out loud, Im not sure if we could have a PR open via CI which generates the change log and replaces the previous?
